python/tk_multi_workfiles/file_save_form.py

```diff
@@ -68,13 +67,19 @@ def __init__(self, parent=None):
             preview_colour = font_colour.darker(140)
         self._preview_colour = (preview_colour.red(), preview_colour.green(), preview_colour.blue())
 
+        self._initializing = True
```

Comment:
I did try to get this to behave correctly by ordering everything inside do_init but the fact it wasn't is a good indication that was a fragile approach.  One thing I would possibly say is that naming it more specifically might make more sense - e.g. call it self._allow_preview_update or something as it's only used for that purpose.